styles/typography.less

```diff
@@ -18,3 +18,40 @@ a {
     color: @primary;
   }
 }
+
+/* Font-size for pages' h1 header; overwriting Bootstrap */
+
+div.page-header h1 {
+  font-size: @page-header-size;
+}
+
+@media (min-width: 992px) {
+  div.page-header h1 {
+    font-size: @page-header-size + @page-header-size/10;
+  }
+}
+
+/* Font-size for buttons; overwriting Bootstrap */
+a.btn {
+  font-size: @link-size;
+}
+
+/* Font-size for page content paragraphs */
+.page-content p {
+  font-size: 1.1rem;
+}
+
+/* Styling font-size for all headers;
+ * every next `h*` element will be smaller then previous in order
+ */
+
+.headerSize(@h_count) when (@h_count < 7) {
+  div.container h@{h_count}, div.container div.h@{h_count} {
```

Comment:
Why use `div.container` here?

Also, can you produce an example screenshot of how this looks to get @sssoz's approval for the style of six levels of headings? In some cases, a designer may wish to use something other than font size to create the heading hierarchy.

Reply:
On this issue will wait for your decision.

Reply:
`.container` is a really broad selector. It effectively means every single row in a Bootstrap grid. It'd be good to find a more specific selector, since many end-users will customize this theme further, and the less specific our selectors the better.

Did you have specific headings in mind that you wanted to override? I might be able to advise on some more limited selectors for those. For example, `/pages/page-common.less` has heading styles for all six headings. Would that be an appopriate place for the headings you're trying to overwrite?

Reply:
Thanks. Done!